framework/py/flwr/cli/run/run.py

```diff
@@ -212,7 +212,8 @@ def _run_with_control_api(
             if is_remote_app:
                 typer.secho(
                     "❌ Failed to start run. Please check that the provided "
-                    "app identifier (@user_name/app_name) is correct.",
+                    "app identifier (@user_name/app_name) is correct, "
```

Comment:
Given the recent [renaming](https://flower.ai/docs/framework/ref-changelog.html#incompatible-changes) to "account" throughout the codebase, I think it's better to use `account_name` here. Wdyt?

```suggestion
                    "app identifier (@account_name/app_name) is correct, "
```